cfme/utils/ext_auth.py

```diff
@@ -30,47 +27,29 @@ def setup_external_auth_ipa(**data):
         iparealm: Realm.
         credentials: Key of the credential in credentials.yaml
     """
-    connect_kwargs = {
-        'username': credentials['host_default']['username'],
-        'password': credentials['host_default']['password'],
-        'hostname': data['ipaserver'],
-    }
-    current_appliance = get_or_create_current_appliance()
-    appliance_name = 'cfmeappliance{}'.format(fauxfactory.gen_alpha(7).lower())
-    appliance_address = current_appliance.hostname
-    appliance_fqdn = '{}.{}'.format(appliance_name, data['iparealm'].lower())
-    with SSHClient(**connect_kwargs) as ipaserver_ssh:
-        ipaserver_ssh.run_command('cp /etc/hosts /etc/hosts_bak')
-        ipaserver_ssh.run_command("sed -i -r '/^{}/d' /etc/hosts".format(appliance_address))
-        command = 'echo "{}\t{}" >> /etc/hosts'.format(appliance_address, appliance_fqdn)
-        ipaserver_ssh.run_command(command)
-    with current_appliance.ssh_client as ssh:
-        result = ssh.run_command('appliance_console_cli --host {}'.format(appliance_fqdn)).success
-        if not current_appliance.is_pod:
-            assert result
-        else:
+    # connect_kwargs = {
```

Comment:
Why leave this commented out when it will be there in previous versions of the file?

Reply:
oh this is still very much a WIP :)